File: app/src/main/java/com/shuoye/video/pojo/Tag.java
package com.shuoye.video.pojo;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.experimental.Accessors;

/**
 * TODO
 *
 * @author shuoye
 * @program video
 * @ClassName Tag
 * @create 2021-10-17 22:46
 **/
@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Accessors(chain = true)
public class Tag {
	private Integer id;
	private Integer animeInfoId;
	private String name;
}

File: app/src/main/java/com/shuoye/video/database/pojo/Tag.kt
package com.shuoye.video.database.pojo

import androidx.room.Entity
import androidx.room.PrimaryKey

/**
 * TODO
 * @program Video
 * @ClassName Tag
 * @author shuoye
 * @create 2021-10-23 17:18
 **/
@Entity(tableName = "tag")
data class Tag(
    @PrimaryKey
    val id: Int,
    val animeInfoId: Int,
    val name: String,
)
